### DISCUSSION TITLE
How to rollback from version 1.1 to 1.0?

### DISCUSSION THREAD

**Discussion by luojunjie2020** [Mikubill/sd-webui-controlnet#1466] [Q&A] [answered]
I accidentally upgraded and didn't backup, and now I can't reproduce the effect I had before. Can I roll back to version 1.0?

**Comment by catboxanon** [comment, marked as answer]
https://github.com/lllyasviel/webui-controlnet-v1-archived

**Reply by luojunjie2020** [reply]
Thanks，you've been a great help.

**Comment by lllyasviel** [comment]
hello you can use old versions in the section "Archive of Deprecated Versions" in Readme or use the link above.
Nevertheless, if you can provide examples that cannot be reproduced with the 1.1 plugin for us to debug, we will highly appreciate that since we are in need of these data - all our tests passed and all our tested previous results can be reproduced.
